components/tidb_query_vec_executors/src/index_scan_executor.rs

```diff
@@ -210,15 +226,12 @@ impl IndexScanExecutorImpl {
         }
     }
 
-    fn process_kv_pair_new(
+    fn extract_columns_from_row(
```

Comment:
maybe call it `extract_columns_from_row_format`

Reply:
Done